### PR TITLE
network-score: Updates to match specs of ONDC

### LINKED ISSUE #411: Update network-score pallet to match specifications
The network-score pallet requires changes to match the specifications (https://dhiway.github.io/confidex/#/flow). Specifically, the structure RatingInputEntry needs to be modified. The following changes are necessary:

Remove: entity_type, provider_uid, entity_uid
Add: entity_id, entity_name, provider_id.
Additionally, all necessary modifications in lib.rs, test.rs, and types.rs are required to implement these changes effectively.

Changes Required
   1. Modify the structure RatingInputEntry as described above.

   2. Update lib.rs to reflect the changes in the structure.
   
   3. Update types.rs to maintain consistency with the modified structure.

   4. Adjust test.rs to accommodate the updated structure and ensure comprehensive testing.

This issue tracks the implementation of these changes to align the network-score pallet with the specified requirements.

### PR DESCRIPTION
**Pull request by vatsa287**
Fixes: #411

### CODE REVIEW

**Review by amarts — COMMENTED** [review]
LGTM

I am in dual mind to add `entity_name` in the chain as `entity_id` is already present. Can we remove that? @smohan-dw any thoughts?

This change is happening due to the API changes requested on the confidex side.

**Comment by amarts** [comment]
@vatsa287 also, whenever the `argument` to the calls (extrinsics) change in pallet, can we do a version number change in runtime? That way, there are no conflicts in APIs.

**Comment by vatsa287** [comment]
> @vatsa287 also, whenever the `argument` to the calls (extrinsics) change in pallet, can we do a version number change in runtime? That way, there are no conflicts in APIs.

Ack, will change to `9105`.

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -127,7 +127,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("cord"),
 	impl_name: create_runtime_str!("dhiway-cord"),
 	authoring_version: 0,
-	spec_version: 9100,
+	spec_version: 9105,
```
Comment (smohan-dw) [inline runtime/src/lib.rs]: Make it 9200, which stands for 0.9.2

**Review by smohan-dw — DISMISSED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (vatsa287) [reply runtime/src/lib.rs]: Updated

**Review by amarts — APPROVED** [review]
LGTM

**Pull request merged**